### ISSUE TITLE
Operational error "no such column: dictionary_language.name_es"

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#643]
Almost all menu items and some admin options lead to the above error in Catalan Signbank -- including the Language menu in the admin where I thought I might add ES.

**Comment by Woseseltops** [comment]
Okay I thinked I fixed it. 

**The problem** 
Signbank wanted to look up the Spanish and Catalan names of TranslationLanguages (not sure why exactly), because Spanish and Catalan were defined as interface languages in the settings. However, the database was not aware of these new interface languages and so had no place to store the TranslationLanguage names in these language. 

To make it very concrete, if you are using Catalan as interface language, Signbank needs to know that the name for TranslationLanguage Catalan is 'Català', but in the database it could only store the name for the TranslationLanguage in English.

**The solution**
This functionality is provided by the Django plugin model-translation. This was introduced by @vanlummelhuizen , so I didn't really know the details of how it worked, but I found Django command `sync_translation_fields` which worked exactly as I expected.

**The cause**
I don't understand how it is possible that everything worked when I finished adding the new pages, but not when Onno looked at it. My best guess is that I did a restart of the process just before I stopped working on the Catalan Signbank, and that this accidentally was the first restart after the interface languages were added to the settings.

**Future prevention**
I've added it as  bullet point in #641
